vuln/core/53.json

```diff
@@ -0,0 +1,16 @@
+{
+  "cve": [
+    "CVE-2018-7166"
+  ],
+  "vulnerable": "10",
+  "patched": ">= 10.9.0",
+  "publish_date": "16-08-2018",
```

Comment:
@vdeturckheim do we have a `publish_date` in the current format?
It should be `y-m-d` if we do.